internal/server/ranking.go

```diff
@@ -63,6 +63,9 @@ var StatsRanking = map[RankKey]int{
 	{"NASA_NEXGDDP", "NASA_Mean_CCSM4", "P1M"}:      0, // IPCC
 	{"NASA_NEXDCP30_StatVarSeriesAggr", "*", "P1M"}: 0, // IPCC
 	{"NASA_NEXGDDP_StatVarSeriesAggr", "*", "P1M"}:  0, // IPCC
+
+	{"NASA_WetBulbComputation_Aggregation", "dcAggregate/NASA_Mean_HadGEM2-AO", "*"}: 0, // Wet bulb year aggregation
+	{"NASA_WetBulbComputation", "dcAggregate/NASA_Mean_HadGEM2-AO", "*"}:             1, // Wet bulb
```

Comment:
do we want to explicitly prefer the new observationPeriod (if available) ?

```
// Prefer HadGEM2 which has most >35'C days across places
{"NASA_WetBulbComputation_Aggregation", "dcAggregate/NASA_Mean_HadGEM2-AO", "P77M"}: 0,
{"NASA_WetBulbComputation_Aggregation", "dcAggregate/NASA_Mean_HadGEM2-AO", "*"}: 1
{"NASA_WetBulbComputation", "dcAggregate/NASA_Mean_HadGEM2-AO", "P77M"}: 0,
{"NASA_WetBulbComputation", "dcAggregate/NASA_Mean_HadGEM2-AO", "*"}: 1,
```

Reply:
spoke offline - since the aggregation only has P78Y, it's okay to omit